lib/rzo/app/templates/Vagrantfile.erb

```diff
@@ -0,0 +1,54 @@
+# -*- mode: ruby -*-
+# vim:ft=ruby
+
```

Comment:
A line with the version of rzo, something like `This file was generated with Rizzo v0.1.0 - https://github.com/ghoneycutt/rizzo`